trestlebot/cli/root.py

```diff
@@ -0,0 +1,30 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2024 Red Hat, Inc.
+
+
+"""Main entrypoint for trestlebot"""
+
+import click
+
```

Comment:
Note @hbraswelrh to add the import for the `create_cmd` from the `create.py` file so that `root.py` knows what to access.